### DISCUSSION TITLE
作者番茄小说的小说不能看了，请处理一下

### DISCUSSION THREAD

**Discussion by 13316282905** [xiaohucode/yidaRule#123] [Ideas]
![Screenshot_2024-03-19-19-18-09-825_com xinqiu yidaapp](https://github.com/xiaohucode/yidaRule/assets/159050612/ea024315-c586-41d3-8bde-95b9a0618a49)
就像这样，请恢复，处理一下

**Comment by 13316282905** [comment]
作者小说的规则好多都看不了，青羽小说和七猫等等的规则都用不了请处理一下

**Comment by lucien-gh** [comment]
是的，阅读app书源也解析失败了；没想到这里也解析失败了

**Comment by 13316282905** [comment]
作者益达app的小说规则不能看的还是不能看，番茄小说的内容也还是不能看

**Comment by Alone4869** [comment]
的确看不了
